### ISSUE TITLE
[BUG] brand-link without background for fixed navbar and some sidebar variants

### ISSUE THREAD

**Issue by Shidersz** [ColorlibHQ/AdminLTE#3534]
**Describe the bug**
The `a.brand-link` element do not get a background color when using a **fixed navbar** layout and a not standard variant color for the sidebar.

**To Reproduce**
1. Go to https://adminlte.io/themes/v3/pages/layout/fixed-topnav.html
2. Open the right sidebar control.
3. Choose dark/light sidebar variant to `lightblue`, `teal`, `olive`, etc.

**Expected behavior**
The `a.brand-link` element should get a background, as when you setup dark/light sidebar variant to `primary`, `success`, etc.

Problem may be fixed with **CSS** support to colors like: `lightblue`, `teal`, `olive`, `lime`, etc. Like the next ones currently available on the package.

```css
.layout-navbar-fixed .wrapper .sidebar-dark-primary .brand-link:not([class*="navbar"]) {
  background-color: #343a40;
}

.layout-navbar-fixed .wrapper .sidebar-light-primary .brand-link:not([class*="navbar"]) {
  background-color: #fff;
}
```

**Screenshots**

Issue show when selecting **dark sidebar variant = lime**

![Screenshot_2021-03-24 AdminLTE 3 Fixed Navbar Layout](https://user-images.githubusercontent.com/63609705/112389164-0da77b00-8cd3-11eb-923d-e172b4c5134f.png)
![Screenshot_2021-03-24 AdminLTE 3 Fixed Navbar Layout(1)](https://user-images.githubusercontent.com/63609705/112389305-4b0c0880-8cd3-11eb-9bfd-dd0550baea53.png)

Issue not show when selecting **dark sidebar variant = primary**

![Screenshot_2021-03-24 AdminLTE 3 Fixed Navbar Layout(2)](https://user-images.githubusercontent.com/63609705/112389332-565f3400-8cd3-11eb-9586-7183c9b85695.png)

**Environment (please complete the following information):**
- AdminLTE Version: v3.1.0
- Operating System: Not relevant
- Browser (Version): Not relevant

**Comment by danny007in** [comment]
@Shidersz u can try to create PR for this

**Comment by Shidersz** [comment]
@danny007in I'm not sure how to proceed, since I'm not familiarized with this package, maybe adding next lines to `_layout.scss` can help, but not 100% sure:

```scss
// Sidebar variants brand-link fix for colors
@each $name, $color in $colors {
  .sidebar-dark-#{$name} .brand-link:not([class*="navbar"]) {
    background-color: $sidebar-dark-bg;
  }

  .sidebar-light-#{$name} .brand-link:not([class*="navbar"]) {
    background-color: $sidebar-light-bg;
  }
}
```

**Comment by danny007in** [comment]
@Shidersz thanks for ur help, fixed in [`d998e4c` (#3542)](https://github.com/ColorlibHQ/AdminLTE/pull/3542/commits/d998e4c167bbd1f5db624950fabb6d65716ec187) 👍🏼
